modules/core/src/lib/attribute-manager.js

```diff
@@ -247,7 +247,8 @@ export default class AttributeManager {
     this.attributeTransitionManager.update({
       attributes: this.attributes,
       numInstances,
-      transitions
+      transitions,
+      context
```

Comment:
Can we pick what we need from context, instead of just forwarding it? It can create a lot of coupling down the road if it is misused.